package.json

```diff
@@ -7,14 +7,20 @@
     "dev": "nuxt dev",
     "generate": "nuxt generate",
     "preview": "nuxt preview",
-    "postinstall": "nuxt prepare"
+    "postinstall": "nuxt prepare",
+    "lint": "eslint --fix './**/*.{js,vue}'"
   },
   "dependencies": {
     "nuxt": "^3.13.0",
     "vue": "latest",
     "vue-router": "latest"
   },
   "devDependencies": {
-    "sass": "^1.79.4"
+    "@eslint/js": "^9.12.0",
+    "eslint": "^9.12.0",
+    "eslint-plugin-vue": "^9.28.0",
+    "globals": "^15.10.0",
+    "sass": "^1.79.4",
+    "typescript-eslint": "^8.8.1"
```

Comment:
Is it not better to use tailwindcss?

Reply:
For me it is, but this is sass is the established styling tech for this repo. Can't change it unless the repo owner consents to it

Reply:
hmm. i know the guy who created the design with figma he likes sass. One could discuss this in another issue, resolving this